101-servicebus-create-namespace/azuredeploy.parameters.json

```diff
@@ -7,6 +7,9 @@
     },
     "serviceBusSku": {
```

Comment:
remove this and use defaultValue in the template

Reply:
resolved